pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Organization management
+---
+
+# Organization management
+
+Organizations in balenaCloud are based around the idea of easier collaboration and robust management of members and applications. The feature blends in with current workflows all the while improving what is already there.
```

Comment:
Agree with Paulo on above. We want a bit more info on what they are and what they will be used for up front

Reply:
I agree @pdcastro the introduction paragraph was something I was holding off for later in the view of plucking a good definition out of the blog John is writing. But, yes this is needed and extremely important for folks to read when they land on the page. I will keep this open, and post what we are adding for a definition.

Reply:
I like the core of what Paulo wrote, but we should add more of the "Why" to the description. For example, "orgs make it easier for teams of developers to collaborate across one or many applications", "orgs allow you to create better distinctions between development and production projects)", "orgs allow more specific access and views of project and codes to help meet security and regulatory rules", etc.

Reply:
@jtonello  You got it, should we also link back to the blog post. Since that covers a lot of the why and other questions that might pop in people's head when going through the feature.

Reply:
![Screenshot from 2020-12-04 13-13-35](https://user-images.githubusercontent.com/22801822/101135926-8bffe480-3632-11eb-82bd-2caa9f086237.png)

How's this looking @pdcastro @shaunmulligan @jtonello Will push this as well.